bigquery_etl/shredder/bug1751979.py

```diff
@@ -370,3 +370,128 @@
 BUG_1751979_MAIN_SUMMARY_V4_REPLACE_CLAUSE = r"""
   REPLACE (sanitize_search_counts_ms(search_counts) AS search_counts)
 """
+
+BUG_1751979_CLIENTS_DAILY_V6_REPLACE_CLAUSE = r"""
+REPLACE (
+  sanitize_search_counts_ms(search_counts) AS search_counts,
+  sanitize_scalar(search_content_urlbar_sum) AS search_content_urlbar_sum,
+  sanitize_scalar(search_content_urlbar_handoff_sum) AS search_content_urlbar_handoff_sum,
+  sanitize_scalar(
+    search_content_urlbar_searchmode_sum
+  ) AS search_content_urlbar_searchmode_sum,
+  sanitize_scalar(search_content_searchbar_sum) AS search_content_searchbar_sum,
+  sanitize_scalar(search_content_about_home_sum) AS search_content_about_home_sum,
+  sanitize_scalar(search_content_about_newtab_sum) AS search_content_about_newtab_sum,
+  sanitize_scalar(search_content_contextmenu_sum) AS search_content_contextmenu_sum,
+  sanitize_scalar(search_content_webextension_sum) AS search_content_webextension_sum,
+  sanitize_scalar(search_content_system_sum) AS search_content_system_sum,
+  sanitize_scalar(search_content_tabhistory_sum) AS search_content_tabhistory_sum,
+  sanitize_scalar(search_content_reload_sum) AS search_content_reload_sum,
+  sanitize_scalar(search_content_unknown_sum) AS search_content_unknown_sum
+)
+"""
+
+BUG_1751979_SEARCH_CLIENTS_DAILY_V8_UDFS = r"""
+CREATE TEMP FUNCTION sanitize_engine_source(
+  engine STRING, `source` STRING
+) AS ((
+    WITH base AS (
```

Comment:
This logic is again copied from the existing `sanitize_search_counts_ms` UDF and refactored to apply to a flat `engine` and `source` input rather than a repeated array of values. Since `search_clients_daily` is basically clients_daily unnested on `search_counts`.